modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -210,39 +176,30 @@ RequestHeader unset X-Remote-User
 
 [discrete]
 [id="requestheader-master-ca-config_{context}"]
-===== Configuring the master
+== Configuring the master
 
-The `identityProviders` stanza in the
-*_/etc/origin/master/master-config.yaml_* file must be updated as well:
+Update the `identityProviders` stanza in the
+`/etc/origin/master/master-config.yaml` file:
```

Comment:
I see I did not answer your question fully, but there really is no such file. See how to configure identity providers in 4.2 here (and modify that module if you find it lacking crucial information to understand how to perform the configuration): https://docs.openshift.com/container-platform/4.2/authentication/identity_providers/configuring-request-header-identity-provider.html#add-identity-provider_configuring-request-header-identity-provider

Reply:
Got it; I see it's specified in the Custom Resource. I'll make that clear.